EDU-14051 - Feedback #4300 - Improvement (#1761)
vtexdocs/dev-portal-content: docs/guides/vtex-io/Storefront-Guides/vtex-io-cross-border-stores/vtex-io-documentation-creating-robots-files-for-cross-border-stores.md

slug: "vtex-io-documentation-creating-robots-files-for-cross-border-stores"
hidden: false
createdAt: "2020-09-15T12:24:38.963Z"
updatedAt: "2025-02-28T13:33:38.731Z"
---

In this guide, you'll learn how to develop and release your own **Robots app**, which manages your cross-border store's ´robots.txt´ files.

A `robots.txt` file controls which folders or files search engines can crawl on a website. Given its impact on a store’s SEO, the following steps explain how to create `robots.txt` files for cross-border stores.

> >⚠️ If your store isn't a cross-border one, check this [link](https://help.vtex.com/tutorial/google-search-console-tracking-robots-txt--tutorials_574?locale=en) to learn how to create a `robots.txt` file through your store's admin.

Reviewer: ```suggestion
>❗ This guide applies only to cross-border stores. If your store is not cross-border, refer to [this guide](https://help.vtex.com/tutorial/google-search-console-tracking-robots-txt--tutorials_574?locale=en).
```

> 

Reviewer: ```suggestion

## `robots.txt` file structure
```

See below the basic structure of a `robots.txt` file:

- `User-agent`: Specifies the search engine robot to which the following rules apply. To apply the same rules to all robots, use `*`.
- `Disallow`:  Defines the paths (relative to the root directory) of files or folders that the search engine specified in `User-agent` must not index.
- `Allow`: Specifies the paths of files within a `Disallowed` folder that are allowed to be crawled and indexed.

You must adjust these directives according to your scenario.

> >⚠️ This feature is available for stores using `vtex.edition-store@3.x` [Edition App](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app/) or newer. To check which Edition App is installed on your account, run `vtex edition get`. If a different Edition is installed, [open a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk) to the VTEX Support asking for the installation of the `vtex.edition-store@5.x`.

Reviewer: ```suggestion
## Before you begin

This feature is available for stores using `vtex.edition-store@3.x` [Edition App](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app/) or newer. To check which Edition app is installed on your account, run `vtex edition get`. If your store is using an older Edition, [open a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk) with VTEX Support asking for the installation of the `vtex.edition-store@5.x`.
```
